packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -207,6 +204,34 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     makeLiquidityPoolKit(shareMint, privateArgs.storageNode),
   );
 
+  const feedKit = zone.makeOnce('Feed Kit', () => makeFeedKit());
+
+  const poolAccountV =
+    // cast to HostInterface
+    /** @type { Vow<HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>>} */ (
+      /** @type {unknown}*/ (
+        zone.makeOnce('Pool Local Orch Account', () => makeLocalAccount())
+      )
+    );
+  const poolAccount = await vowTools.when(poolAccountV);
```

Comment:
Reviewers - I'm under the impression this async work is safe to do here since we've already defined all kinds?

Reply:
yes; see the ` // ^^^ Define all kinds above this line. Keep remote calls below. vvv` comment around 181.